client/conf/server.properties.in

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 # The binding interface for the management server
```

Comment:
What about a further documentation/comment here describing why this variable is commented, and the default behavior?

Reply:
Will fix this.

Reply:
Will fix this.

Reply:
fixed